index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"><img src="images/image_geometry_1.png" alt="logo image"></div>
+        <nav>
+            <div class="Button">≡</div>
+        </nav>
+    </header>
+    <main>
+        <h1 class="intro">
+            Javed <br> Shahbaz
+        </h1>
+        <section class="paragraph">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+         </section>    
+    </main>
+    <section class="cButton",background: #fff;>
+        <img src="images/Start collaboration.png" alt="Start collaboration">
+    </section>
+    <div class="image">
+        <img src="images/image_geometry_2.png" alt="image">
+    </div>
```

Comment:
- [ ] This should not be used as link but button or a tag so that it can be clickable. Also remeber to set the hover and others.